cucumber/authenticators_oidc/features/authn_status_oidc.feature

```diff
@@ -216,6 +216,59 @@ Feature: OIDC Authenticator - Status Check
     Then the HTTP response status code is 500
     And the authenticator status check fails with error "RequiredResourceMissing: CONJ00036E"
 
+  Scenario: service-id missing and a 500 error response is returned
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-oidc
+      body:
+      - !webservice
+
+      - !webservice
+        id: status
+        annotations:
+          description: Status service to verify the authenticator is configured correctly
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: id-token-user-property
```

Comment:
Are these 2 variables needed?  Been a while since I looked at policy for OIDC...

Reply:
yes, these are the mandatory variables for oidc